### ISSUE TITLE
The poor baseline with Res50

### ISSUE THREAD

**Issue by PkuRainBow** [CSAILVision/semantic-segmentation-pytorch#83]
I use your default settings to train the C1-Bilinear-Deepsup and only achive not very good performance,

`
Mean IoU: 0.3418, Accuracy: 76.33%`

I admit that we can reproduce the PSP baseline with your method, but it seems that there exists large gap between your results and the reported baseline in the paper of PSP.

**Comment by Tete-Xiao** [comment]
I'd encourage you to go through the PSPNet paper more carefully to find out what tricks or techniques we didn't use for this toolbox.
